mne/io/eyelink/eyelink.py

```diff
@@ -284,6 +287,45 @@ def _find_overlaps(df, max_time=0.05):
     return ovrlp.drop(columns=tmp_cols).reset_index(drop=True)
 
 
+@fill_doc
+def read_eyelink_calibration(
+    fname, screen_size=None, screen_distance=None, screen_resolution=None
+):
+    """Return info on calibrations collected in an eyelink file.
+
+    Parameters
+    ----------
+    fname : str
+        Path to the eyelink file (.asc).
+    screen_size : tuple of shape (2,)
+        The width and height (in meters) of the screen that the eyetracking
+        data was collected with. For example ``(.531, .298)`` for a monitor with
+        a display area of 531 x 298 mm. Defaults to None.
+    screen_distance : float
+        The distance (in meters) from the participant's eyes to the screen.
+        Defaults to None.
+    screen_resolution : tuple of shape (2,)
+        The resolution (in pixels) of the screen that the eyetracking data
+        was collected with. For example, ``(1920, 1080)`` for a 1920x1080
+        resolution display. Defaults to None.
+
+    Returns
+    -------
+    calibrations : list
+        A list of :class:`mne.preprocessing.eyetracking.Calibration` instances, one for
+        each eye of every calibration that was performed during the recording session.
+    """
+    fname = Path(fname)
```

Comment:
I believe we actually have a convenience function for this, which also checks if the file exists. Must be somewhere in `utils`... `check_fname` or so? I don't remember

Reply:
I saw that! I was going to ask if we could use that. I can try using that instead and push a commit.

Reply:
Probably better to use `name='fname'` because that's the name of the argument in your function

Reply:
Thanks both - I incorporated this and @hoechenberger 's other suggestions in [5015c9b](https://github.com/mne-tools/mne-python/pull/11719/commits/5015c9b4e581fef4428d01cc5e203f0cdf131904)